backend/tournesol/entities/base.py

```diff
@@ -30,6 +31,24 @@ def filter_date_lte(cls, qs, dt):
     def filter_date_gte(cls, qs, dt):
         return qs.filter(add_time__gte=dt)
 
+    @classmethod
+    def filter_metadata(cls, qst, filters):
+        for fil in filters:
+            key, values = fil
+
+            # add several OR expressions if multiple values are provided
+            if len(values) > 1:
+                expression = Q(**{"metadata__" + key: values[0]})
+
+                for val in values[1:]:
+                    expression = expression | Q(**{"metadata__" + key: val})
+
+                qst = qst.filter(expression)
```

Comment:
Would an `__in` operation work here out of the box, instead of the multiple `|` ("or") operations?

Reply:
I'm going to test. It's possible that the generated query runs faster with `IN` rather than with `OR`.